Kentico.Widget.RichText/Models/Widgets/RichTextWidgetViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Kentico.Components.Web.Mvc.Widgets.Models
+{
+    /// <summary>
+    /// Rich text widget view model.
+    /// </summary>
+    public class RichTextWidgetViewModel
+    {
+        /// <summary>
+        /// Name of the widget property to edit.
+        /// </summary>
+        public string PropertyName { get; set; }
```

Comment:
Quite weird property... First look is like WTF is that..maybe I would choose different name.